packages/BUILD.bazel

```diff
@@ -818,6 +818,15 @@ latex_package(
     tests = ["tikz_test.tex"],
 )
 
+latex_package(
+    name = "tikz-qtree",
+    srcs = [
+        "@texlive_texmf__texmf-dist__tex__latex__tikz-qtree",
+        ":pgf",
```

Comment:
The same holds for this commit, switch the internal and external sources around to adhere to the style used throughout this file.

Reply:
It may be the case that `buildifier` will auto-sort this for you.

Reply:
Thanks! Done.